src/test/resources/keycloak-import/sling.json

```diff
@@ -1237,7 +1285,7 @@
   "internationalizationEnabled" : false,
   "supportedLocales" : [ ],
   "authenticationFlows" : [ {
-    "id" : "bd07af7f-436a-4c7d-a2ae-78e20a897cb3",
+    "id" : "aff611f8-41d9-49ad-ba2b-7a24ae030bc8",
```

Comment:
I think this (and the following ID changes) are not necessary. It would be better to exclude them so we don't pollute the history.

Reply:
I did several changes on the exported keycloak configuration:

1. Added a new client that support pkce authentication
2. Modified the user profile to have few attributes
3. Defined few groups, added the test user as member and added them to the claim returned to the clients